Ball of Duty Server/Domain/Game.cs

```diff
@@ -16,7 +16,13 @@ public void AddPlayer(Player player, string clientIp, int clientPort)
         {
             _players.Add(player.Id, player);
             player.CurrentCharacter = Map.AddCharacter(); // TODO data to character creation should be dynamic
-            Map.Broker.AddTarget(player.Id, clientIp, clientPort);
+
+            double charX = player.CurrentCharacter.Body.Position.X;
+            double charY = player.CurrentCharacter.Body.Position.Y;
+            double charWidth = player.CurrentCharacter.Body.Width;
+            double charHeight = player.CurrentCharacter.Body.Height;
+            Map.Broker.AddPlayer(player.Id, player.CurrentCharacter.Id, charX, charY, charWidth, charHeight, clientIp,
```

Comment:
preferably give the Broker the character instead or a DAO. or we could let Map do this in Map.AddCharacter()